### DISCUSSION TITLE
Delete User Directory after User Deletion

### DISCUSSION THREAD

**Discussion by Joshua2504** [drakkan/sftpgo#1384] [Q&A]
Hi,

I'm trying to setup the [Event Manager](https://github.com/drakkan/sftpgo/blob/main/docs/eventmanager.md) to delete a users home directory if the user is deleted.

I created the following event rule.

<img width="1604" alt="image" src="https://github.com/drakkan/sftpgo/assets/11087078/dc6e82fd-8786-49bc-8162-0b35a5c12445">

This is my event action:

<img width="830" alt="image" src="https://github.com/drakkan/sftpgo/assets/11087078/45b74dfb-2abf-4643-8dd4-8284671e216d">

This does nothing, even if I set the full path to delete manually.

Could this be what I need?

<img width="561" alt="image" src="https://github.com/drakkan/sftpgo/assets/11087078/6805ee93-0282-4970-b7d1-99c008bb2291">

I also tried setting filers to *, without success.

Any help is appreciated.

**Comment by Joshua2504** [comment]
Hi,

any help with this is highly appriciated.

Regards

**Reply by macduyhai** [reply]
You can use API for this

**Reply by teknatha136** [reply]
> Hi,
> 
> any help with this is highly appriciated.
> 
> Regards

works with `-rf,/path/to/dir/{{ObjectName}}`

**Reply by fhufenreuter** [reply]
> > Hi,
> > any help with this is highly appriciated.
> > Regards
> 
> works with `-rf,/path/to/dir/{{ObjectName}}`

thanks für the hint, this did the trick. But is there also a placeholder for home dir? As not always the {{ObjectName}} is the same as homedir.
